nielsen/src/nielsen/NOLBUNDLE.ts

```diff
@@ -1,40 +1,104 @@
 /* eslint-disable */
-import { NielsenOptions } from './Types';
+import { NielsenCountry, NielsenOptions } from './Types';
 
-export function loadNielsenLibrary(appId: string, instanceName: string, options?: NielsenOptions) {
-    // https://engineeringportal.nielsen.com/docs/DTVR_Browser_SDK
-    // Add Static Queue Snippet to initialize a Nielsen SDK Instance.
-    // @ts-ignore
-    !(function (t: Window, n: any) {
-        t[n] = t[n] || {
-            nlsQ: function (e: any, o: any, c?: any, r?: any, s?: any, i?: any) {
-                // @ts-ignore
-                return (
-                    (s = t.document),
-                    (r = s.createElement('script')),
-                    (r.async = 1),
-                    (r.src =
-                        ('http:' === t.location.protocol ? 'http:' : 'https:') +
-                        '//cdn-gl.imrworldwide.com/conf/' +
-                        e +
-                        '.js#name=' +
-                        o +
-                        '&ns=' +
-                        n),
-                    (i = s.getElementsByTagName('script')[0]),
-                    i.parentNode.insertBefore(r, i),
-                    (t[n][o] = t[n][o] || {
-                        g: c || {},
-                        ggPM: function (e: any, c: any, r?: any, s?: any, i?: any) {
-                            // @ts-ignore
-                            (t[n][o].q = t[n][o].q || []).push([e, c, r, s, i]);
-                        }
-                    }),
-                    t[n][o]
-                );
+export function loadNielsenLibrary(
+    appId: string,
+    instanceName: string,
+    options?: NielsenOptions,
+    country?: NielsenCountry
+) {
+    if (country == NielsenCountry.CZ) {
```

Comment:
If I understand correctly CZ country should correlate to DCR and otherwise we have DTVR. Can they ever be used at the same time? 
Should we prefer the check of DTVR/DCR as this is used in the other parts of the code? Or would we need a check that chosen country and DCR/DTVR is possible?

Not important but the two snippets below seem very similar. Could this be simplified/combined in some way?

Reply:
For now I think the implementation allows all combinations of possible values for `dtvrEnabled`, `dcrEnabled` and `country` are possible. However, at the moment I think DTVR is US only (see my other reply on your Types.ts comment). To be confirmed.

Indeed the snippets look very similar. The CZ Nielsen rep instructed me to use separate snippets as per the documentation for each country, just in case. I also asked the US based Nielsen rep if there's any chance to use a single snippet.  

The only function we use in the integration is `ggPM`, which all the snippets I could find in the engineering portal offer and the endpoint to report to is always the same. The only obvious differences I spotted were variable names, surrounding try catch blocks with error logging in the CZ snippet and some snippets that don't have the `trackEvent` function. Also, within the same integration guide pages, the snippet is the same for DCR and DTVR (if present).